.github/actions/spelling/advice.txt

```diff
@@ -0,0 +1,27 @@
+<!-- See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples%3A-advice --> <!-- markdownlint-disable MD033 MD041 -->
+<details><summary>If you see a bunch of garbage</summary>
```

Comment:
This content is included as supplemental text to a GitHub comment when a commit/PR introduces unknown words into the repository in a file that isn't excluded on a line where that content isn't masked.